repo/packages/Q/quobyte/0/marathon.json.mustache

```diff
@@ -0,0 +1,39 @@
+{
+  "id": "{{quobyte.framework-name}}",
+  "cpus": {{quobyte.cpus}},
+  "mem": {{quobyte.mem}},
+  "instances": {{quobyte.instances}},
+  "args": [
+    "/opt/quobyte-mesos-cmd.sh",
+    "--zk","{{quobyte.zk}}",
+    "--master", "{{mesos.master}}",
+    "--port", "{{quobyte.port}}",
+    "--docker_image", "{{quobyte.docker_image}}",
```

Comment:
will this ever be different than `framework_image`? if so, it probably needs an entry in resources.json